charts/aiven-operator-crds/templates/aiven.io_cassandras.yaml

```diff
@@ -139,8 +139,8 @@ spec:
                   maxLength: 128
                   type: string
                 project:
-                  description: Target project.
-                  format: ^[a-zA-Z0-9_-]*$
+                  description: Identifies the project this resource belongs to
+                  format: ^[a-zA-Z0-9_-]+$
```

Comment:
This can't be an empty string, right?

Reply:
Can't imagine why project name would be empty. The field is required, though.